patterns/cta-newsletter.php

```diff
@@ -11,24 +11,28 @@
  */
 
 ?>
-<!-- wp:cover {"tagName":"aside","overlayColor":"accent-1","isUserOverlayColor":true,"minHeight":460,"isDark":false,"metadata":{"categories":["call-to-action"],"patternName":"twentytwentyfive/newsletter-sign-up","name":"Newsletter Sign Up"},"align":"full","style":{"spacing":{"padding":{"right":"var:preset|spacing|50","left":"var:preset|spacing|50","top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"}}},"layout":{"type":"constrained","contentSize":"800px"}} -->
-<aside class="wp-block-cover alignfull is-light" style="padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50);min-height:460px"><span aria-hidden="true" class="wp-block-cover__background has-accent-1-background-color has-background-dim-100 has-background-dim"></span><div class="wp-block-cover__inner-container">
-	<!-- wp:heading {"textAlign":"center","fontSize":"xx-large"} -->
-	<h2 class="wp-block-heading has-text-align-center has-xx-large-font-size"><?php esc_html_e( 'Sign up to get daily stories', 'twentytwentyfive' ); ?></h2>
-	<!-- /wp:heading -->
+<!-- wp:group {"tagName":"aside","metadata":{"categories":["call-to-action"],"patternName":"twentytwentyfive/cta-newsletter","name":"Newsletter Sign Up"},"align":"full","className":"is-style-section-3","style":{"spacing":{"padding":{"right":"var:preset|spacing|50","left":"var:preset|spacing|50","top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"}},"dimensions":{"minHeight":""}},"layout":{"type":"constrained","contentSize":"800px"}} -->
```

Comment:
Nit: remove metadata.

Reply:
I believe we need to keep the categories so that the shuffle functionality can be more robust.

Have we concluded whether we're keeping the pattern name and name? I saw some discussions going on.

Reply:
But the category is listed in the file header.
Who can give a definite answer?

Reply:
I could be getting it wrong, but it seems that the suffle functionality relies on the metadata. Please check [this code](https://github.com/WordPress/gutenberg/blob/03cfbff40ef2e5bf9ad369bb0a18642aa70833ef/packages/block-editor/src/components/block-toolbar/shuffle.js#L34).

Reply:
The metadata inserted in the editors when you insert the pattern.
So it is not missing from the shuffle functionality. 

I went down a rabbit hole trying to find any official documentation, dev note or similar.

Reply:
Sounds good, thanks for that. Let's try to define what we do with the metadata in the issue and work on it in bulk.